spec/models/ability_spec.rb

```diff
@@ -227,6 +228,7 @@
       is_expected.to be_able_to(:manifest, flagged_scanned_resource)
       is_expected.to be_able_to(:color_pdf, color_enabled_resource)
       is_expected.to be_able_to(:read, :graphql)
+      is_expected.to be_able_to(:download, other_staff_file)
```

Comment:
can you use a different resource name here? the reference to staff file implies that maybe it's not public.

Reply:
This resource is used several places in the tests, and the name is to contrast with `staff_file` where we test staff interacting with their own files vs. with files created by another staff member. 

I'd be happy to use `admin_file` instead, or create a new file called `open_file` if that seems like a clearer illustration.

Reply:
Yes I think `open_file` would be clearer in this context!

Reply:
👍 i've pushed another commit that switches to `open_file`